tests/unit/coordinator/plugins/cassandra/test_plugin.py

```diff
@@ -7,52 +7,53 @@
 from astacus.coordinator.plugins.base import StepFailedError
 from astacus.coordinator.plugins.cassandra import plugin
 from astacus.coordinator.plugins.cassandra.model import CassandraConfigurationNode
+from pytest_mock import MockerFixture
 from tests.unit.conftest import CassandraTestConfig
-from types import SimpleNamespace
+from unittest.mock import Mock
 
 import pytest
 
 
 @pytest.fixture(name="cplugin")
-def fixture_cplugin(mocker, tmpdir):
-    ctc = CassandraTestConfig(mocker=mocker, tmpdir=tmpdir)
-    yield plugin.CassandraPlugin(
-        client=ctc.cassandra_client_config, nodes=[CassandraConfigurationNode(listen_address="127.0.0.1")]
+def fixture_cplugin(cassandra_test_config: CassandraTestConfig) -> plugin.CassandraPlugin:
+    return plugin.CassandraPlugin(
+        client=cassandra_test_config.cassandra_client_config,
+        nodes=[CassandraConfigurationNode(listen_address="127.0.0.1")],
     )
 
 
 @pytest.mark.asyncio
-async def test_step_cassandrasubop(mocker):
+async def test_step_cassandrasubop(mocker: MockerFixture) -> None:
     mocker.patch.object(plugin, "run_subop")
 
     step = plugin.CassandraSubOpStep(op=ipc.CassandraSubOp.stop_cassandra)
-    cluster = None
-    context = None
-    result = await step.run_step(cluster, context)
+    cluster = Mock()
+    context = Mock()
+    result = await step.run_step(cluster, context)  # type: ignore[func-returns-value]
     assert result is None
```

Comment:
The test disables the static check for "returns None" and then assert that the return is None.
I think we can just remove the assert and the ignore.